npm/css-src/sass/main.scss

```diff
@@ -14,3 +14,41 @@
   .hide {
   display: none;
 }
+
+.drag-and-drop {
+  margin-bottom: 40px;
+}
+
+.drag-and-drop__dropzone {
+  display: flex;
+  outline: 3px dashed #0b0c0c;
```

Comment:
I think you can replace this hex value with `$govuk-input-border-colour` because the colour is in the [design system palette](https://design-system.service.gov.uk/styles/colour/).

Reply:
Thanks! I'll bookmark that link too.